src/tools/pub/cmd/pub-outdated.md

```diff
@@ -201,36 +201,55 @@ so that each package uses the versions in the **Resolvable** column.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
-`--json`
-: Use this option to generate output in JSON format.
+### `--json`
+
+Generates output in JSON format.
+
+### `--[no-]color`
+
+Controls whether the output uses color for emphasis.
+The default depends on whether you're using this command at a terminal.
+
+At a terminal, `--color` is the default;
+otherwise, `--no-color` is the default.
```

Comment:
I like how you've made the option descriptions start with -s verbs. I'm not crazy about having so many short paragraphs in a row, though. And I think we could be clearer about `no-` behavior.